lib/server.js

```diff
@@ -92,6 +92,22 @@ class Server {
     } = prepareApplications(applications));
   }
 
+  // Update application version for each connection.
+  // It will update to a newest version in the requested range. If client
+  // requested version in (1.1) range he will not be updated to (1.2.x). If no
+  // suitable version is found client will use it's previous app version.
```

Comment:
Missing a comma and an article here: `'...is found, the client will...'`.
Also, `'it's'` shouldn't have an apostrophe, it is a possessive here.